src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java

```diff
@@ -22,12 +22,14 @@
 
 import org.apache.commons.httpclient.InvalidRedirectLocationException;
 import org.apache.commons.httpclient.URIException;
+import org.apache.commons.lang3.StringUtils;

```

Comment:
This class/library is not bundled with core (`org.apache.commons.lang.StringUtils` is).